youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -175,7 +175,7 @@ def _ffmpeg_filename_argument(self, fn):
         # Always use 'file:' because the filename may contain ':' (ffmpeg
         # interprets that as a protocol) or can start with '-' (-- is broken in
         # ffmpeg, see https://ffmpeg.org/trac/ffmpeg/ticket/2127 for details)
-        return 'file:' + fn
+        return 'file:' + fn if fn != '-' else fn
```

Comment:
adding `- used to pipe the media stream to the standard output` to the comment.
is this good as an explanation?